01_dask.delayed.ipynb

```diff
@@ -29,7 +29,7 @@
   },
```

Comment:
<p>graphs / DAGs should probably be singular.</p><p> </p><p>Slight issue with using "efficient", since it's meaning is overloaded. I'd stick to something like "Calling the delayed versions is much faster because the original functions haven't been called yet."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dask/dask-tutorial/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I agree.  I will address these.